doc/interfaces/data_fetch.rst

```diff
@@ -0,0 +1,23 @@
+-------------------------
+Downloading DIPY datasets
+-------------------------
+In this tutorial, you will learn how to download the DIPY datasets using the terminal.
+
+First, let's create a folder::
+
+    mkdir data_folder
+
+You can list all available datasets in DIPY using the command::
+
+    dipy_fetch list
+
+In order to fetch a specific dataset, you will need to specify its name to the ``dipy_fetch``
+command, and an optional destination directory using the ``--out_dir`` argument::
+
+    dipy_fetch {specific_dataset} --out_dir {data_folder}
```

Comment:
`{specific_data_out_folder}`  instead of `{data_folder}`

It will avoid any confusion with the example below.

Reply:
Addressed in   df5173f, b0e5af2